templates/blog/post_detail.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+
+{% block content %}
+<h1>{{ post.title }}</h1>
+{{ post.owner.username }} at {{ post.created_time }}
+<p>{{ post.content }}</p>
+
+{{ post.comments.count }} comment{{ post.comments.count|pluralize }}
+<div class="comment-section">
+  {% if comments %}
+  {% for comment in post.comments.all %}
```

Comment:
The iteration over `post.comments.all` should be `comments` as per the context provided in `get_context_data`. Change `{% for comment in post.comments.all %}` to `{% for comment in comments %}` to match the context variable.